moped-database/migrations/1713466319581_moped_proj_tags_audit_fields/up.sql

```diff
@@ -0,0 +1,24 @@
+-- Adds audit columns and trigger to moped_proj_tags
+ALTER TABLE moped_proj_tags
+ADD COLUMN created_at TIMESTAMPTZ NOT NULL DEFAULT now(),
+ADD COLUMN created_by_user_id INT4 NULL,
+ADD COLUMN updated_by_user_id INT4 NULL,
```

Comment:
super minor, but i think it is conventional to declare `integer` instead of `int4`. even though they are effectively identical i think `integer` is SQL standard?

Reply:
gotcha! tbh was copying one of mikes audit PRs for some of this code hehe. also wasnt sure about which columns should be nullable or not, does this look right?

Reply:
in an ideal world, we would backfill these columns so that we could make them `not null` going forward, but we've never discussed it. what you have looks good to me!

Reply:
ahhhh i created a copypasta cascade 🍝  fyi - i wondered if it originally stemmed from Hasura, but they also declare with `integer` when you add from the console so that further supports John's thoughts here. thanks for flagging this!

Also, nullable sounds good to me since we won't be able backfill those values for previous project tags.